### ISSUE TITLE
quickpick checkbox state not announced correctly by screen readers

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#210671]
1. enable a screen reader
2. run the `Manage Trusted Extensions` command.
3. checkboxes are always read as "unchecked"

**Comment by TylerLeonhardt** [comment]
bisected this to c2f0a3fdf3b202d91f0a7765eb0785ea8b61bf39 @hediet can you take a look?

**Comment by hediet** [comment]
Oops, @TylerLeonhardt next time we meet I owe you a drink! ;)
